authors/urls.py

```diff
@@ -46,4 +48,7 @@
     path('api/v1/', include('authors.apps.profiles.urls'), name='apiv1'),
     path('api/v1/', include('authors.apps.articles.urls')),
     path('api/v1/', include('authors.apps.rating.urls')),
+    path('api/v1/', include('authors.apps.comments.urls')),
+    path('api/v1/', include('authors.apps.rating.urls')),
+    path('api/v1/', include('authors.apps.comments.urls')),
```

Comment:
You have two similar paths ```path('api/v1/', include('authors.apps.comments.urls'))```
 kindly remove one cause there the same

Reply:
Thanks, I'm correcting that